ETT-Backend/Models/EmployeeResponse.cs

```diff
@@ -14,11 +14,15 @@ public class EmployeeResponse
         [JsonProperty("growth")]
         public GrowthHours Growth;
 
+        [JsonProperty("dbUpdate")]
+        public LastUpdatedAt DBUpdate;
+
         public EmployeeResponse(Employee emp)
         {
             EmployeeId = emp.EmployeeNumber;
             Billable = new BillableHours(emp.CurrentBillableHours, emp.BillableTargetToDate, emp.YearlyBillableTargetHours);
             Growth = new GrowthHours(emp.CurrentTrainingHours, emp.TargetTrainingHours - emp.CurrentTrainingHours, emp.TargetTrainingHours);
+            DBUpdate = new LastUpdatedAt(emp.UpdatedAt);
```

Comment:
According to my comment, UpdatedAt is a fine name..
[JsonProperty("updatedAt")]
public String UpdatedAt;
...
UpdatedAt = emp.UpdatedAt.ToString("MM/dd/yyyy HH:mm:ss");